### PR TITLE
etcdmain: quota-backend-bytes in help.go

### PR DESCRIPTION
**Pull request by gyuho**

### CODE REVIEW

**Comment by gyuho** [comment]
/cc @heyitsanthony 

It would be good, if we use CLI package to automate this, like `cobra`.

**Comment by heyitsanthony** [comment]
@gyuho I think this flag should be listed under the `member flags` heading instead of `experimental`.

/cc @xiang90 -- any opinion?

**Comment by gyuho** [comment]
Just moved to `member flags`. I agree that it should be in `member flags`, not as `experimental`.

File: etcdmain/help.go [hunk 1]
```diff
@@ -51,7 +51,8 @@ member flags:
 		maximum number of wal files to retain (0 is unlimited).
 	--cors ''
 		comma-separated whitelist of origins for CORS (cross-origin resource sharing).
-
+	--quota-backend-bytes '0'
+		raise alarms when backend size exceeds the given quota. 0 means use the default quota.
```
Comment (heyitsanthony) [inline etcdmain/help.go]: `(0 defaults to low space quota)` to be consistent with the other descriptions?

**Comment by heyitsanthony** [comment]
lgtm

**Comment by gyuho** [comment]
Thanks. Will merge in greens.

On Thu, Apr 21, 2016, 13:07 Anthony Romano <notifications@github.com> wrote:

> lgtm
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/etcd/pull/5146#issuecomment-213093789>
>
-- 
Sincerely,
Gyu-Ho Lee

**Pull request merged**